Mathlib/LinearAlgebra/RootSystem/Defs.lean

```diff
@@ -380,6 +372,31 @@ lemma pairing_reflection_perm_self_right (i j : ι) :
     sub_add_cancel_left, ← toLin_toPerfectPairing, map_neg, toLin_toPerfectPairing,
     root_coroot_eq_pairing]
 
+/-- The indexing set of a root pairing carries an involutive negation, corresponding to the negation
+of a root / coroot. -/
+def indexNeg : InvolutiveNeg ι where
```

Comment:
This doesn't seem to be used in the rest of the PR. Do you have later applications in mind? And should there be a `@[simps]` to generate something like `indexNeg_apply`?

Reply:
Good point, added `@[simps]`.

I do have later applications in mind (for use in proofs, not statements). I admit I _might_ not end up using it but I still think it is worth having.